src/mesh.cpp

```diff
@@ -235,6 +259,28 @@ bool loadPlyFile(const QString& fileName,
     }
     if (info.colors.size() != info.verts.size())
         info.colors.clear();
+
+    // Reconstruct inner polygons
+    while (innerPolygonVertexCount.size())
```

Comment:
This would be cleaner as a for loop - no need to be modifying `innerPolygonVertexCount`, right?  (If you needed to do it this way, the conditional check would also be better written as `while(!innerPolygonVertexCount.empty())`)